test/push-metadata/expected-stdout

```diff
@@ -0,0 +1,12 @@
+%A
+DEBUG	Project state has been successfully loaded.
+DEBUG	Change description: "my change description"
+DEBUG	Validation done.
+INFO	Plan for "push" operation:
+INFO	  * C foo/extractor/ex-generic-v2/without-rows | changed: metadata
+INFO	  × C main/extractor/ex-generic-v2/empty
+DEBUG	Execution plan is valid.
+%A
+INFO	Push done.
+%A
+DEBUG	Sent "push" successful event id: "%s"
```

Comment:
Tento test by nemusel byt `verbose`, ak tu netestujes nejake `debug` hlasky.

Reply:
Vacsinou je `verbose` iba jeden test, napr, `push` a ostatne su `non-verbose`.
Zbytocne je potom potrebne prepisovat vela testov, ak sa zmeni nejaka debug hlaska.

Reply:
Aha, upravím. To jsem se asi trefil, jeden jsem náhodně vzal a zkopíroval. 🙂